### PR TITLE
Create container to display datahub messages

### PR DESCRIPTION
**Pull request by GusBaamonde**
Create container to display the datahub messages.
This container will allow us to continue implementing the designs of the messages.

PR realated with:  https://github.com/FromDoppler/doppler-webapp/pull/151

datahub-messages

### CODE REVIEW

File: assets/scss/templates/_reportes.scss [hunk 1]
```diff
@@ -1,3 +1,27 @@
+
+.dp-datahub-messages {
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: What do you think about _dp-rich-message_?

File: assets/scss/templates/_reportes.scss [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+    margin: $dp-spaces--lv0;
+    font-weight: $dp-font-weight-bold;
+    color: $dp-color-grey;
+  }
+
+  p {
+    margin: $dp-spaces--lv0 $dp-spaces--lv0 $dp-spaces--lv2;
+  }
+
+  .dp-datahub-action {
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: Again, what do you think about remove datahub from here?

File: assets/scss/templates/_reportes.scss [hunk 3]
```diff
@@ -81,7 +105,7 @@
 /* Section KPI */
 .container-reports {
   width: 100%;
-  padding: 60px 0;
+  padding: 60px 2%;
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: percentage? is it fine?

Should you not use the levels?

File: assets/scss/core/_variables.scss [hunk 4]
```diff
@@ -14,7 +14,7 @@ $dp-font-small-lineheight: 18px;
 $dp-font-weight: 400 !default;
 $dp-font-family: Arial, sans-serif !default;
 $dp-font-family-proximanova: "proxima-nova", Helvetica;
-$dp-font-size: 16px !default;
+$dp-font-size: 15px !default;
```
Comment (andresmoschini) [inline assets/scss/core/_variables.scss]: Is there not a font size level?

File: assets/scss/templates/_reportes.scss (hunk 3 above)
Comment (andresmoschini) [reply assets/scss/templates/_reportes.scss]: I saw that you have changed it by levels, but, I am still curious, is it the same? are levels not absolute and percentage relative? how are they equivalent?

Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: El porcentaje es relativo y los niveles son absolutos, solo tome algo proporcional. no es lo mismo. se uso porcentaje para que sea relativo al tamaño de pantalla. Pero como planteamos niveles no estaría mal utlizarlos.

File: assets/scss/core/_variables.scss (hunk 4 above)
Comment (GusBaamonde) [reply assets/scss/core/_variables.scss]: This case is an exception. The levels we have jump from 6 to 6. 12 is very small and 18 is too big. The mockups have the size in 15. I preferred to adjust to the design. In any case, this size is within a variable and can be modified.

Comment (andresmoschini) [reply assets/scss/core/_variables.scss]: I is fine for me, thanks for the clarification!

**Comment by andresmoschini** [comment]
I think that it is a nice step in the right direction. Thanks @GusBaamonde !

:shipit:

**Pull request merged**